python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -141,6 +141,12 @@ class DetectAndMeasureConfig(pipeBase.PipelineTaskConfig,
         target=SkyObjectsTask,
         doc="Generate sky sources",
     )
+    badSourceFlags = lsst.pex.config.ListField(
+        dtype=str,
+        doc="Do not include sources with any of these flags set in the output catalog.",
```

Comment:
I just realized my rewording could be misinterpreted (they're not "set in the output catalog"!). Maybe:

```suggestion
        doc="Sources with any of these flags set are not included in the output catalog.",
```